app/assets/javascripts/admin/addon/components/dashboard-new-feature-item.hbs

```diff
@@ -1,17 +1,16 @@
 <div class="admin-new-feature-item">
-  <div class="new-feature-emoji">{{this.item.emoji}}</div>
   <div class="new-feature-content">
     <div class="header">
-      {{#if this.item.link}}
-        <a
-          href={{this.item.link}}
-          target="_blank"
-          rel="noopener noreferrer"
-        >{{this.item.title}}</a>
-      {{else}}
-        {{this.item.title}}
+      <h3>{{this.item.title}}</h3>
```

Comment:
We are missing the link for the new feature too, maybe the title can link to it? I will bring it up in the internal topic too.

![image](https://github.com/discourse/discourse/assets/920448/9f53591d-a7d3-4c1d-b523-184a79a8aee1)